go_lib/dependency/requirements/store.go

```diff
@@ -47,3 +47,13 @@ func (m *MemoryValuesStore) Get(key string) (interface{}, bool) {
 	v, ok := m.values[key]
 	return v, ok
 }
+
+func (m *MemoryValuesStore) GetAll() interface{} {
```

Comment:
Why `interface{}` not `map[string]interface{}`

Reply:
Apply